modules/study/src/main/StudySocket.scala

```diff
@@ -59,142 +59,136 @@ final private class StudySocket(
     case RP.In.TellRoomSri(studyId, P.In.TellSri(sri, user, tpe, o)) =>
       import Protocol.In.{ *, given }
       import JsonView.given
-      def who = user map { Who(_, sri) }
+      def who = user.map(Who(_, sri))
+      def applyWho(f: Who => Unit) = who.foreach(f)
```

Comment:
This function is to avoid repetation of `who.foreach(f)`. But I'm not so sure about the name. Really appreciate a name suggestion.

Reply:
:shrug: